client/lib/plugins/wporg-data/actions.js

```diff
@@ -54,6 +57,18 @@ var PluginsDataActions = {
 			searchTerm: searchTerm
 		} );
 
+		if ( 'featured' === category ) {
+			this.fetchCuratedList( 'featured', [
+				'woocommerce',
+				'jetpack',
+				'bbpress',
+				'buddypress',
+				'akismet',
+				'vaultpress',
```

Comment:
My idea was to have `curated.json` be a data store of plugins that could be added to over time if needed, while this array says which ones you want to use from that store. I think it would be easier to manage over time this way (just switching a slug here, rather than always having to mess with `curated.json`), but maybe that's overkill?